administrator/language/en-GB/plg_system_compat.ini

```
@@ -5,5 +5,7 @@
 
 PLG_COMPAT_FIELD_CLASSES_ALIASES_LABEL="Classes Aliases"
 PLG_COMPAT_FIELD_CLASSES_ALIASES_DESCRIPTION="Add class aliases for classes which have been renamed or moved to a namespace."
+PLG_COMPAT_FIELD_ES5_ASSETS_LABEL="ES5 Assets"
+PLG_COMPAT_FIELD_ES5_ASSETS_DESCRIPTION="Activate this option in case your extension requires *.es5 assets which lead to an exception. The assets provided are empty but prevent the exception."
```

Now I understand it - how is this?

```suggestion
PLG_COMPAT_FIELD_ES5_ASSETS_DESCRIPTION="Activate this option if your extension requires *.es5 assets which has resulted in an exception. The assets provided are empty but prevent the exception."
```